packaging/rpm/microshift.spec

```diff
@@ -208,6 +211,9 @@ fi
 %post selinux
 
 %selinux_modules_install -s %{selinuxtype} %{_datadir}/selinux/packages/%{selinuxtype}/microshift.pp.bz2
+if /usr/sbin/selinuxenabled ; then
+    %microshift_relabel_files
```

Comment:
oh, so we need to do it on %post on the system and not on %install? 

Then we'd want to remove 196? 

Can we label it even with selinux not enabled just in case it's enabled afterwards?

Reply:
Yep good call will remove